OptimizelySDK/OptimizelyUserContext.cs

```diff
@@ -90,15 +89,25 @@ public OptimizelyDecision Decide(string key)
         {
             return Optimizely.Decide(this, key, options);
         }
- 
+
+        /// <summary>
+        /// Returns a key-map of decision results for multiple flag keys and a user context.
+        /// </summary>
+        /// <param name="keys">list of flag keys for which a decision will be made.</param>
+        /// <returns>A dictionary of all decision results, mapped by flag keys.</returns>
+        public Dictionary<string, OptimizelyDecision> DecideForKeys(List<string> keys, List<OptimizelyDecideOption> options)
+        {
+            return Optimizely.DecideForKeys(this, keys, options);
+        }
+
         /// <summary>
         /// Returns a key-map of decision results for multiple flag keys and a user context.
         /// </summary>
         /// <param name="keys">list of flag keys for which a decision will be made.</param>
         /// <returns>A dictionary of all decision results, mapped by flag keys.</returns>
         public Dictionary<string, OptimizelyDecision> DecideForKeys(List<string> keys)
         {
-            throw new NotImplementedException();
+            return Optimizely.DecideForKeys(this, keys, new List<OptimizelyDecideOption>());
```

Comment:
Call the local DecideForKeys(keys, new List<OptimizelyDecideOption>()) to be consistent with others?